docs/usage/configuration-options.md

```diff
@@ -1556,6 +1556,18 @@ To adjust it down to 10s for all queries, do this:
 }
 ```
 
+### certificateAuthority
+
+certificateAuthority overrides trusted CA certificate.
+
+### privateKey
```

Comment:
The GitHub Blog has an article about mTLS authentication, and how developers can help keep things safe: [^1]

> In this post, we'll deep dive into some interesting attacks on mTLS authentication. We'll have a look at implementation vulnerabilities and how developers can make their mTLS systems vulnerable to user impersonation, privilege escalation, and information leakages.

[^1]: https://github.blog/2023-08-17-mtls-when-certificate-authentication-is-done-wrong/